doc/contribute.rst

```diff
@@ -0,0 +1,252 @@
+
+-  Follow the coding-guidelines_ as for scikit-learn.
+
+-  When applicable, use the validation tools and other code in the
+   `ramp_utils` subpackage.
+
+-  If your pull request addresses an issue, please use the title to describe
+   the issue and mention the issue number in the pull request description to
+   ensure a link is created to the original issue.
+
+-  All public methods should have informative docstrings with sample
+   usage presented as doctests when appropriate.
+
+-  Please prefix the title of your pull request with `[MRG]` if the
+   contribution is complete and should be subjected to a detailed review.
+   Incomplete contributions should be prefixed `[WIP]` to indicate a work
+   in progress (and changed to `[MRG]` when it matures). WIPs may be useful
+   to: indicate you are working on something to avoid duplicated work,
+   request broad review of functionality or API, or seek collaborators.
+   WIPs often benefit from the inclusion of a `task list`_ in the PR description.
+
+-  All other tests pass when everything is rebuilt from scratch. On
+   Unix-like systems, check with (from the toplevel source folder)::
+
+        $ make
+
+-  Documentation and high-coverage tests are necessary for enhancements
+   to be accepted.
+
+-  At least one paragraph of narrative documentation with links to
+   references in the literature (with PDF links when possible) and
+   the example.
+
+.. _coding-guidelines: http://scikit-learn.org/dev/developers/contributing.html#coding-guidelines
+.. _task list: https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments
+
+You can also check for common programming errors with the following
+tools:
+
+-  Code with good unittest coverage (at least 80%), check with::
+
+        $ pip install pytest pytest-cov
+        $ pytest -vsl .
+
+-  No flake8 warnings (which includes pep8 and pyflakes), check with::
+
+        $ pip install flake8
+        $ flake8 path/to/module.py
+
+Filing bugs
+===========
+We use Github issues to track all bugs and feature requests; feel free to
+open an issue if you have found a bug or wish to see a feature implemented.
+
+It is recommended to check that your issue complies with the
+following rules before submitting:
+
+-  Verify that your issue is not being currently addressed by other
+   issues_ or `pull requests`_.
+
+-  Please ensure all code snippets and error messages are formatted in
+   appropriate code blocks.
+   See `Creating and highlighting code blocks`_.
+
+-  Please include your operating system type and version number, as well
+   as your Python, scikit-learn, numpy, and scipy versions. This information
+   can be found by runnning the following code snippet::
+
+    python
+    import platform; print(platform.platform())
+    import sys; print("Python", sys.version)
+    import numpy; print("NumPy", numpy.__version__)
+    import scipy; print("SciPy", scipy.__version__)
+    import sklearn; print("Scikit-Learn", sklearn.__version__)
```

Comment:
Here we just need to know the version of the different ramp packages instead of numpy, scipy, and scikit-learn

Reply:
@glemaitre Thanks for your comments. I am not sure if I understand this wone. What versions of ramp packages?